### DISCUSSION TITLE
Custom "Page Not Found"

### DISCUSSION THREAD

**Discussion by ghost** [facebook/docusaurus#5079] [Q&A] [answered]
Ola

Is this page customisable?

![image](https://user-images.githubusercontent.com/50320942/123688798-c214e800-d852-11eb-83db-10a01265b97a.png)

I would like to place one additional information there 🤔

I believe I'd need to swizzle something but can not figure out what. Thanks for a pointer 👈

**Comment by idontknowjs** [comment, marked as answer]
Hey @dominikmeyersap yes you can! 
Try out `npm run swizzle <theme name> NotFound`
